dBay integration

LuisEGV opened the pull request:
Added Feature: I added the integration for dBay, a dApp marketplace (https://www.dbay.ai/).

The changes were based on the dApp integration documentation: https://status.im/developer_tools/add_your_dapp.html

Commit:
dBay integration

ghost commented:
Hey @LuisEGV, and thank you so much for making your first pull request in status-react! :heart: Please help us make your experience better by filling out this brief questionnaire https://goo.gl/forms/uWqNcVpVz7OIopXg2

<blockquote><img src="https://lh5.googleusercontent.com/wpBxRKA0YohehKsTaptQsaYJNVcNA9m2IOLvxXpSMcamdM8e4q7gyYBQ_yRVvNH-Ez8=w1200-h630-p" width="48" align="right"><div><img src="//ssl.gstatic.com/docs/forms/device_home/android_192.png" height="14"> Google Docs</div><div><strong><a href="https://docs.google.com/forms/d/e/1FAIpQLSewRMgMD6FLe9yO7oCipVn7YTn1HboJc4ZDJ_Va_BsDAemQwQ/viewform?usp=send_form&usp=embed_facebook">Status 1st commit dev survey</a></strong></div><div>First of all thank you so much for taking the time to make your first foray into the world of Status. We want to make sure that your experience with building the future of the decentralized web with us is not only enjoyable but gives you real purpose.

 Please check out our wiki at https://wiki.status.im and our Dev forum at https://discuss.status.im for more in depth longform discussion -- You can also join our Riot at https://chat.status.im which has all of our developers to answer any pressing questions you may have.</div></blockquote>

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status.html))?

Commit:
Merge branch 'develop' into develop

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/2/) | 2019-02-10 08:14:42 | ~15 sec | `android` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/2/) | 2019-02-10 08:14:42 | ~17 sec | `android-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7459/2/) | 2019-02-10 08:14:44 | ~15 sec | `ios-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7459/2/) | 2019-02-10 08:14:45 | ~10 sec | `windows` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7459/2/) | 2019-02-10 08:14:46 | ~14 sec | `linux` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7459/2/) | 2019-02-10 08:14:47 | ~17 sec | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7459/2/) | 2019-02-10 08:14:51 | ~18 sec | `macos` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7459/2/consoleText) |
| | | | | | | |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/3/) | 2019-02-20 11:05:48 | ~15 sec | `android` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/3/) | 2019-02-20 11:05:48 | ~18 sec | `android-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7459/3/) | 2019-02-20 11:05:52 | ~13 sec | `linux` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/linux/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7459/3/) | 2019-02-20 11:05:58 | ~11 sec | `windows` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/windows/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7459/3/) | 2019-02-20 11:05:58 | ~15 sec | `macos` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/macos/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7459/3/) | 2019-02-20 11:06:01 | ~25 sec | `ios-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios-e2e/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#3](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7459/3/) | 2019-02-20 11:06:02 | ~25 sec | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7459/3/consoleText) |
| :x: | e1871ef6 | [#4](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/4/) | 2019-03-05 17:41:09 | ~15 sec | `android-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/4/consoleText) |
| :x: | e1871ef6 | [#4](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/4/) | 2019-03-05 17:41:10 | ~14 sec | `android` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/4/consoleText) |
| :x: | e1871ef6 | [#4](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7459/4/) | 2019-03-05 17:41:14 | ~16 sec | `ios` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-7459/4/consoleText) |

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/2/) | 2019-02-10 08:14:42 | ~15 sec | `android` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7459/2/consoleText) |
| :x: | a9bd3900 | [#2](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/2/) | 2019-02-10 08:14:42 | ~17 sec | `android-e2e` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-7459/2/consoleText) |

oskarth commented:
Not sure why this PR was left hanging for 10 days, pinged people in dapp team. Sorry about the delayed response @LuisEGV!

rachelhamlin commented on src/status_im/ui/screens/browser/default_dapps.cljs:
```diff
@@ -16,6 +16,10 @@
              :dapp-url    "https://app.ercdex.com/"
              :photo-path  "contacts://erc-dex"
              :description "Trustless trading has arrived on Ethereum"}
+             {:name        "dBay"
```
Perhaps dBay would be a better fit under `Marketplaces`?

rachelhamlin commented on src/status_im/ui/screens/browser/default_dapps.cljs:
```diff
@@ -16,6 +16,10 @@
+             {:name        "dBay"
+             :dapp-url    "https://www.dbay.ai/"
+             :photo-path  "contacts://dBay"
+             :description "Buy all Yours dApps in one place."}
```
Suggested copy tweak: `Buy from all your favorite DApps in one place.` You guys trade ERC721s, right, @LuisEGV?

LuisEGV replied:
I just changed this commit to fit the description you suggested and also included dBay in the marketplace section. Thanks for that Rachel :)

rachelhamlin replied:
Thanks for the changes @LuisEGV!

rachelhamlin commented:
Thanks @LuisEGV! I left a few comments.

Commit:
Moved dBay to Marketplace and changed description

I moved dBay to the Marketplace section and changed the description to "Buy from all your favorite DApps in one place" based on @Rachel Hamlin awesome suggestion.

LuisEGV commented:
Hey, guys (@rachelhamlin @flexsurfer ) are their things that I need to change for this commit to go through. @jeluard is there something you want me to change?

flexsurfer commented:
@LuisEGV could you please rebase onto develop and squash commits? thanks

StatusSceptre commented:
@LuisEGV Any update on rebase and squash commits?

LuisEGV commented:
> @LuisEGV Any update on rebase and squash commits?
Hi StatusSpectre! I am having some difficulties with rebasing to my first commit because I committed with different accounts. Is there any possibility you guys could go over the last three commits and see if it will work rather than squashing.

jeluard commented:
@LuisEGV You could recreate a new PR?

LuisEGV commented:
> @LuisEGV You could recreate a new PR?

Yeah, I think that's the best thing to do 👍

LuisEGV commented:
The new pull request is here: https://github.com/status-im/status-react/pull/7706 👍
